python/ml4ir/model/architectures/dnn.py

```diff
@@ -7,17 +7,17 @@ def architecture_op(ranking_features):
         first_dense = layers.Dense(num_nodes, activation="relu", name="first_dense")(
             ranking_features
         )
-        # first_dropout = layers.Dropout(rate=0.3, name="first_dropout")(first_dense)
+        first_dropout = layers.Dropout(rate=0.3, name="first_dropout")(first_dense)
```

Comment:
make configurable?  dropout rate, and if 0, leave off the layer entirely?

Reply:
Okay. I was debating if I should and in turn probably make it more generic than it has to be. But good call, I like configs. Will do.